lib/config.js

```diff
@@ -130,6 +134,10 @@ const conf = convict({
     port: {
       default: '3306',
       env: 'MYSQL_PORT'
+    },
+    connectionLimit: {
```

Comment:
just browsing through the PRs, I could not find where this `connectionLimit` is used